pontoon/static/css/dark-theme.css

```diff
@@ -0,0 +1,68 @@
+/* Dark Theme Variables */
+:root {
+  --black-1: #1c1e21;
+  --black-2: #000000;
+  --black-3: #000;
+  --black-4: #272a2f;
+  --black-5: rgba(0, 0, 0, 0.8);
+  --black-6: black;
+  --black-7: rgb(39, 42, 47);
```

Comment:
That's the same as  [#272a2f](https://www.rgbtohex.net/), which is already defined above.